src/reason-parser/reason_parser.mly

```diff
@@ -1705,6 +1705,10 @@ structure_item:
       EXTERNAL as_loc(val_ident) COLON core_type EQUAL primitive_declaration
       { let loc = mklocation $symbolstartpos $endpos in
         mkstr (Pstr_primitive (Val.mk $3 $5 ~prim:$7 ~attrs:$1 ~loc)) }
+    | item_attributes
+      EXTERNAL as_loc(val_ident) COLON core_type SEMI
+      { let loc = mklocation $symbolstartpos $endpos in
+        mkstr (Pstr_primitive (Val.mk $3 $5 ~prim:[""] ~attrs:$1 ~loc)) }
```

Comment:
I think instead of `~prim:[""]` we may want to add the name in `val_ident`. Especially because BuckleScript has started to produce warnings about `""` in externals, and people will have a hard time figuring out that we're generating it at the AST level.

Reply:
I see... I've just updated the PR, do you mind looking again? :) Thanks for reviewing!

Reply:
I think the new change is a good start -- but now the old version will not print to the new one. I think in the printer you still need the `[""]` case in the match statement

Reply:
I did not realize that `external map: int = "map";` and `external map: int = "";` are the same 🙈 should be ok now, thank you